.github/workflows/ci-base-tests.yml

```diff
@@ -28,7 +28,7 @@ jobs:
           - ./smarts/core
           - ./smarts/env --ignore=./smarts/env/tests/test_rllib_hiway_env.py
           - ./smarts/env/tests/test_rllib_hiway_env.py
-          - ./smarts/sstudio
+          - ./smarts/sstudio --ignore=./smarts/sstudio/genhistories.py
```

Comment:
Why ignore `--ignore=./smarts/sstudio/genhistories.py` a non test file in linux test?